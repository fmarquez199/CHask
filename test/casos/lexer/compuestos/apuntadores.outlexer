(Power), pos (1,1)
(puff), pos (1,9)
Identificador "p", pos (1,14)
(=), pos (1,16)
(summon), pos (1,18)
(Power), pos (1,25)
(\n), pos (1,30)
(Rune), pos (2,1)
(puff), pos (2,9)
Identificador "p", pos (2,14)
(=), pos (2,16)
(summon), pos (2,18)
(Rune), pos (2,25)
(\n), pos (2,29)
(Runes), pos (3,1)
(puff), pos (3,9)
Identificador "p", pos (3,14)
(=), pos (3,16)
(summon), pos (3,18)
(Runes), pos (3,25)
(\n), pos (3,30)
(Battle), pos (4,1)
(puff), pos (4,9)
Identificador "p", pos (4,14)
(=), pos (4,16)
(summon), pos (4,18)
(Battle), pos (4,25)
(\n), pos (4,31)
(Skill), pos (5,1)
(puff), pos (5,9)
Identificador "p", pos (5,14)
(=), pos (5,16)
(summon), pos (5,18)
(Skill), pos (5,25)
(\n), pos (5,30)
(Skill), pos (6,1)
(puff), pos (6,9)
Identificador "p", pos (6,14)
(=), pos (6,16)
(summon), pos (6,18)
(Skill), pos (6,25)
(|}), pos (6,30)
Entero 10, pos (6,32)
({|), pos (6,34)
(\n), pos (6,36)
Identificador de tipo "Contactopuff", pos (7,1)
Identificador "p", pos (7,14)
(=), pos (7,16)
(summon), pos (7,18)
Identificador de tipo "Contacto", pos (7,25)
(\n), pos (7,33)
(Kit), pos (8,1)
(of), pos (8,5)
(Power), pos (8,9)
(puff), pos (8,15)
Identificador "p", pos (8,20)
(=), pos (8,22)
(summon), pos (8,24)
(Kit), pos (8,31)
(of), pos (8,35)
(Power), pos (8,38)
(\n), pos (8,43)
(puff), pos (10,1)
Identificador "p", pos (10,6)
(=), pos (10,8)
Entero 13, pos (10,10)
(\n), pos (10,12)
(free), pos (12,1)
Identificador "p", pos (12,6)
(\n), pos (12,7)
(free), pos (13,1)
(|}), pos (13,5)
({|), pos (13,7)
Identificador "p", pos (13,10)
(\n), pos (13,11)
Identificador "p", pos (15,1)
(==), pos (15,3)
(DeathZone), pos (15,6)
(\n), pos (15,15)