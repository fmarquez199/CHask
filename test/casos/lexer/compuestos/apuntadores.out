Token Power en la fila: 1, columna: 1
Token puff en la fila: 1, columna: 9
Token identificador "p" en la fila: 1, columna: 14
Token = en la fila: 1, columna: 16
Token summon en la fila: 1, columna: 18
Token Power en la fila: 1, columna: 25
Token End Instruction \n en la fila: 1, columna: 30
Token Rune en la fila: 2, columna: 1
Token puff en la fila: 2, columna: 9
Token identificador "p" en la fila: 2, columna: 14
Token = en la fila: 2, columna: 16
Token summon en la fila: 2, columna: 18
Token Rune en la fila: 2, columna: 25
Token End Instruction \n en la fila: 2, columna: 29
Token Runes en la fila: 3, columna: 1
Token puff en la fila: 3, columna: 9
Token identificador "p" en la fila: 3, columna: 14
Token = en la fila: 3, columna: 16
Token summon en la fila: 3, columna: 18
Token Runes en la fila: 3, columna: 25
Token End Instruction \n en la fila: 3, columna: 30
Token Battle en la fila: 4, columna: 1
Token puff en la fila: 4, columna: 9
Token identificador "p" en la fila: 4, columna: 14
Token = en la fila: 4, columna: 16
Token summon en la fila: 4, columna: 18
Token Battle en la fila: 4, columna: 25
Token End Instruction \n en la fila: 4, columna: 31
Token Skill en la fila: 5, columna: 1
Token puff en la fila: 5, columna: 9
Token identificador "p" en la fila: 5, columna: 14
Token = en la fila: 5, columna: 16
Token summon en la fila: 5, columna: 18
Token Skill en la fila: 5, columna: 25
Token End Instruction \n en la fila: 5, columna: 30
Token Skill en la fila: 6, columna: 1
Token puff en la fila: 6, columna: 9
Token identificador "p" en la fila: 6, columna: 14
Token = en la fila: 6, columna: 16
Token summon en la fila: 6, columna: 18
Token Skill en la fila: 6, columna: 25
Token |} en la fila: 6, columna: 30
Token entero 10 en la fila: 6, columna: 32
Token {| en la fila: 6, columna: 34
Token End Instruction \n en la fila: 6, columna: 36
Token identificador "Contactopuff" en la fila: 7, columna: 1
Token identificador "p" en la fila: 7, columna: 14
Token = en la fila: 7, columna: 16
Token summon en la fila: 7, columna: 18
Token identificador "Contacto" en la fila: 7, columna: 25
Token End Instruction \n en la fila: 7, columna: 33
Token Kit en la fila: 8, columna: 1
Token of en la fila: 8, columna: 5
Token Power en la fila: 8, columna: 9
Token puff en la fila: 8, columna: 15
Token identificador "p" en la fila: 8, columna: 20
Token = en la fila: 8, columna: 22
Token summon en la fila: 8, columna: 24
Token Kit en la fila: 8, columna: 31
Token of en la fila: 8, columna: 35
Token Power en la fila: 8, columna: 38
Token End Instruction \n en la fila: 8, columna: 43
Token End Instruction \n en la fila: 9, columna: 1
Token puff en la fila: 10, columna: 1
Token identificador "p" en la fila: 10, columna: 6
Token = en la fila: 10, columna: 8
Token entero 13 en la fila: 10, columna: 10
Token End Instruction \n en la fila: 10, columna: 12
Token End Instruction \n en la fila: 11, columna: 1
Token free en la fila: 12, columna: 1
Token identificador "p" en la fila: 12, columna: 6
Token End Instruction \n en la fila: 12, columna: 7
Token free en la fila: 13, columna: 1
Token |} en la fila: 13, columna: 5
Token {| en la fila: 13, columna: 7
Token identificador "p" en la fila: 13, columna: 10
Token End Instruction \n en la fila: 13, columna: 11
Token End Instruction \n en la fila: 14, columna: 1
Token identificador "p" en la fila: 15, columna: 1
Token == en la fila: 15, columna: 3
Token DeathZone en la fila: 15, columna: 6
Token End Instruction \n en la fila: 15, columna: 15
